tests/iio_attr.c

```diff
@@ -41,7 +41,7 @@
 enum backend {
 	LOCAL,
 	XML,
-	AUTO
+	AUTO,
```

Comment:
will `AUTO` always be the last in the enum?
if so, this is fine;
typically the last element in an enum/series gets a comma, mostly for future-proofing the patches;
i.e. the patch becomes just an addition

Reply:
The trailing comma was to fix some formatting issues that clang-format wanted to see.